ggshield/core/filter.py

```diff
@@ -211,3 +211,35 @@ def censor_content(content: str, policy_breaks: List[PolicyBreak]) -> str:
                 )
             )
     return content
+
+
+def censor_line_content(content: str, policy_breaks: List[PolicyBreak]) -> str:
```

Comment:
It looks like this code is not used. Can you remove it, or is something missing?